scripts/triggers-end-to-end-tests/v2/index.js

```diff
@@ -23,12 +23,18 @@ const AUTH_BLOCKING_CREATE_V2_LOG =
 const AUTH_BLOCKING_SIGN_IN_V2_LOG =
   "========== AUTH BLOCKING SIGN IN V2 FUNCTION METADATA ==========";
 const RTDB_LOG = "========== RTDB V2 FUNCTION ==========";
+const HTTPS_FUNCTION_LOG = "========== HTTPS V2 FUNCTION ==========";
 
 const PUBSUB_TOPIC = "test-topic";
 const START_DOCUMENT_NAME = "test/start";
 
 admin.initializeApp();
 
+exports.httpsv2reaction = functionsV2.https.onRequest((req, res) => {
+  console.log(HTTPS_FUNCTION_LOG);
```

Comment:
this seems unnecessary for http function since the test script can check the status of the http request directly (assuming that we don't need http function calling another http function wrapping)

Reply:
fixed